django/contrib/admin/views/main.py

```diff
@@ -37,7 +37,8 @@
 class ChangeList(object):
     def __init__(self, request, model, list_display, list_display_links,
             list_filter, date_hierarchy, search_fields, list_select_related,
-            list_per_page, list_max_show_all, list_editable, model_admin):
+            list_per_page, list_max_show_all, list_editable, model_admin,
+            orderable_by):
```

Comment:
This argument is what caused most of the tests to fail. Perhaps we can add a default, otherwise all the tests need to be updated.